client/src/leap/soledad/client/api.py

```diff
@@ -209,12 +211,15 @@ def __init__(self, uuid, passphrase, secrets_path, local_db_path,
         self._crypto = SoledadCrypto(self._secrets.remote_storage_secret)
         self._init_u1db_sqlcipher_backend()
 
+        self.syncable = syncable
```

Comment:
I find confusing that `syncable` exist also in `shared_db` do we need it in both places?